src/main/fc/fc_msp.c

```diff
@@ -115,6 +115,9 @@
 #define STATIC_ASSERT(condition, name) \
     typedef char assert_failed_ ## name [(condition) ? 1 : -1 ] __attribute__((unused))
 
+#define BYTES_REMAINING_GTE(src, num) \
```

Comment:
I don't really like the use of a macro here. I think it obscures rather than clarifies the code. I'd rather see (eg)
```
if (sbufBytesRemaining(src) >= 10) {
```
rather than
```
 if (BYTES_REMAINING_GTE(src, 10)) {
```